src/index.ts

```diff
@@ -528,10 +528,10 @@ export const beautifier: Beautifier = {
       //   }
       // ]
     },
-    JSX: true,
-    JavaScript: true
+    JavaScript: true,
+    JSX: true
   },
-  beautify(data) {
+  beautify(data: any) {
```

Comment:
Should be `any`. I think the interface is called `BeautifyData` or something from Unibeautify.